docker/dockerfile.tf

```diff
@@ -12,12 +12,13 @@ FROM ${FULL_IMAGE} as triton
 FROM ${BASE_IMAGE} as base
 
 # Triton TF backends
-COPY --chown=1000:1000 --from=triton /opt/tritonserver/backends/tensorflow2 backends/tensorflow2/
+COPY --chown=1000:1000 --from=triton /opt/tritonserver/backends/tensorflow backends/tensorflow/
```

Comment:
in which triton container is this in `/opt/tritonserver/backends/tensorflow` instead of `.../tensorflow2`?

I've checked 22.11, 23.02, 23.04 and all have it in `/opt/tritonserver/backends/tensorflow2`.